.gitignore

```diff
@@ -7,6 +7,7 @@ PySDM_examples/utils/temporary_files/*
 *.svg
 *.gif
 *.ipynb.badges.md
+*.json
```

Comment:
this might be too general since, e.g., we do store bibliography data in a JSON file in the repo